### DISCUSSION TITLE
ColossalChat tends to defend on its FAKE knowledge and still be authoritative.

### DISCUSSION THREAD

**Discussion by pps43** [hpcaitech/ColossalAI#3336] [Community | General]
The right side is from ChatGPT. I don't quite understand why they give totally different answers on a technique fact (ChatGPT is right because I know PhysX library). I upload the chat history to see if it helps to tune this model better.

![image](https://user-images.githubusercontent.com/14260091/228742505-99c1009a-8f75-4c5c-bc3a-51778983feac.png)

**Comment by FrankLeeeee** [comment]
The current dataset is quite limited, so we can't expect it to perform as well as ChatGPT. As releaved by OpenAI, they spent months on the alignment, which is insufficient on our ColossalChat.
